src/js/components/AirPlayButton.js

```diff
@@ -142,8 +29,121 @@ AirPlayButton = {
  * @see http://docs.videojs.com/module-videojs.html#~registerPlugin
  */
 module.exports = function(videojs) {
```

Comment:
Not necessary as part of this VJS 7+ compatibility pass, but `videojs` should probably be an imported/required dependency instead of needing to be passed in. Mostly mentioning this on this pull request because this change set likely requires a major version bump and it might be a good opportunity to make that change too.